packages/edit-site/src/components/header/index.js

```diff
@@ -79,64 +79,81 @@ export default function Header( {
 	const displayBlockToolbar =
 		! isLargeViewport || deviceType !== 'Desktop' || hasFixedToolbar;
 
+	const { hoveredEntityIds, getBlock } = useSelect( ( select ) => {
+		const { getHoveredEntities, getBlock } = select( 'core/block-editor' );
+
+		return {
+			hoveredEntityIds: getHoveredEntities(),
+			getBlock,
+		};
+	} );
+
+	const hoveredName =
+		getBlock( hoveredEntityIds[ 0 ] )?.attributes.slug || '';
+
 	return (
-		<div className="edit-site-header">
-			<MainDashboardButton.Slot>
-				<FullscreenModeClose />
-			</MainDashboardButton.Slot>
-			<div className="edit-site-header__toolbar">
-				<Button
-					isPrimary
-					isPressed={ isInserterOpen }
-					className="edit-site-header-toolbar__inserter-toggle"
-					onClick={ onToggleInserter }
-					icon={ plus }
-					label={ _x(
-						'Add block',
-						'Generic label for block inserter button'
+		<>
+			<div className="edit-site-header">{ hoveredName }</div>
```

Comment:
Again this just just the temporary ugly thing to show the hovered name for now. 😆